content/lorem.md

```diff
@@ -4,6 +4,7 @@ date  = 2022-02-05
 
```

Comment:
Comment regarding line 3: Change date so it is consistent with last page update.

Reply:
Dates updated in a378f02.